src/OmnichannelChatSDK.ts

```diff
@@ -2158,6 +2173,67 @@ class OmnichannelChatSDK {
         }
     }
 
+    private async setPrechatConfigurations(liveWSAndLiveChatEngJoin: LiveWSAndLiveChatEngJoin) {
```

Comment:
most of these functions seem to be Promise<void> return type, can we declare it explicitly instead of relying on implicit return type? This prevents developers from accidentally changing it.